src/gui/qgsattributeform.cpp

```diff
@@ -262,6 +331,110 @@ bool QgsAttributeForm::save()
   if ( changedLayer )
     mLayer->triggerRepaint();
 
+  return success;
+}
+
+void QgsAttributeForm::resetMultiEdit( bool promptToSave )
+{
+  if ( promptToSave )
+    save();
+
+  setMultiEditFeatureIds( mLayer->selectedFeaturesIds() );
+}
+
+bool QgsAttributeForm::saveMultiEdits()
+{
+  //find changed attributes
+  QMap< int, QVariant > newAttributeValues;
```

Comment:
Should this be a `QgsAttributeMap`? (It doesn't really make a difference, I know...)